openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -944,6 +1040,9 @@ def build_workfile():
     elif task_name in ("lighting", "light", "render", "rendering"):
         build_render(project_name, asset_name)
 
+    elif task_name == "fabrication":
```

Comment:
as `fabrication` is an early step, I think it's better to move it up and replace `build_model` block with this block.